examples/build_sediment.ipynb

```diff
@@ -140,7 +141,8 @@
     "* `./wflow_test_sediment` : output model folder\n",
     "* `\"{'subbasin': [12.2051, 45.8331], 'strord': 4, 'bounds': [11.70, 45.35, 12.95, 46.70]}\"` : derive a subbasin with its outlet at the given x,y coordinates and within the given bounds [xmin, ymin, xmax, ymax] (WGS84) snapped to a river with minimum stream order (strord) of 4. All *REGION* options are described in the [docs](https://deltares.github.io/hydromt/latest/user_guide/model_region)\n",
     "* `-r 0.0083333` : build the model at a 30 arc sec (~1km) resolution\n",
-    "* `-i wflow_sediment_build.ini` : setup configuration file using\n",
+    "* `-i wflow_sediment_build.ini` : model  configuration file\n",
```

Comment:
Just a minor remark, but perhaps it is better to name this `model setup configuration file`, to avoid confusion with the "real" configuration file of the wflow_sediment model.